modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,167 @@
+
+[IMPORTANT]
+====
+Automatic discovery and provisioning is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Use the following procedure to create discovery requests, automatically discover local devices, and automatically provision local volumes for selected devices.
+
+.Prerequisites
+
+* The Local Storage Operator is installed.
+
+* Local disks are attached to {product-title} nodes.
+
+* You have access to the {product-title} web console and the `oc` command-line interface (CLI).
+
+.Procedure
+. To create a local volume discovery request from the web console:
+
+.. Navigate to *Operators* -> *OperatorHub* -> *Installed Operators*.
+
+.. Click on *Local Storage* from the Operators list.
+
+.. Select the *Local Volume Discovery Result* tab to display the list of results.
+
+.. Click *Create Local Volume Discovery Result*.
+
+.. Complete the required fields in *Form View* or click *YAML view* for full control.
+
+.. When finished, click *Create*. The new result is displayed.
+. To display a continuous list of available devices on a node:
+
+.. Log in to the {product-title} web console.
+
+.. Navigate to *Compute* -> *Nodes*.
+
+.. Click the node name that you want to open. The "Node Details" page is displayed.
+
+.. Select the *Disks* tab to display the list of the selected devices.
++
+The device list updates continuously as local disks are added or removed. You can filter the devices by name, status, type, model, capacity, and mode.
++
+[NOTE]
+====
+Results are deleted after they are removed from the node. Symlinks must be manually removed.
+====
+
+. To automatically provision local volumes for the discovered devices from the web console:
+
+.. Navigate to *Operators* -> *Installed Operators* and select *Local Storage* from the list of Operators.
+
+.. Select *Local Volume Set* -> *Create Local Volume Set*.
+
+.. Enter a volume set name and a storage class name.
+
+.. Choose *All nodes* or *Select nodes* to apply filters accordingly.
+
+.. Select the disk type, mode, size, and limit you want to apply to the local volume set, and click *Create*.
++
+A message displays after several minutes, indicating that the "Operator reconciled successfully."
+
+[start=3]
+. Alternately, to provision local volumes for the discovered devices from the CLI:
```

Comment:
Should this be s/Alternately/Alternatively/?

Also, if this is an alternative option to the preceding step, should this step and the preceding one be bullet points? I.e. if they are mutually exclusive then a numbered sequence might not be suitable.

Reply:
Yeah, should be alternatively for sure, since it's a choice. I debated about the numbering because it's not both, but then there are so many ordered sub-tasks within each that I opted for what seemed the most tidy.